src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#3C7680"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#304a7d"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#2c6675"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry.stroke",
+					"stylers": [
+						{
+							"color": "#255763"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#b0d5ce"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "transit.line",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#283d6a"
+						}
+					]
+				},
+				{
+					"featureType": "transit.station",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#3a4762"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#0e1626"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#4e6d70"
+						}
+					]
+				}
+			],
+			{name: 'Night'});
+
+	// Create a map object, and include the MapTypeId to add
+	// to the map type control.
+	var map = new google.maps.Map(document.getElementById('map'), {
+		center: {lat: 1.2764558, lng: 103.7996469},
+		zoom: 16,
+		mapTypeControlOptions: {
+			mapTypeIds: ['roadmap', 'satellite', 'hybrid', 'terrain',
+				'Night']
+		}
+	});
+
+	//Associate the styled map with the MapTypeId and set it to display.
+	map.mapTypes.set('Night', styledMapType);
+	map.setMapTypeId('Night');
```

Comment:
Same for the `'Night'` string here. It's what we call a magic number: https://en.wikipedia.org/wiki/Magic_number_(programming)

We want to avoid having these repeated constants that are defined inline because its a pain to change it in the future, and can easily be missed, resulting in bugs.